core/templates/services/playthrough.service.ts

```diff
@@ -210,32 +208,20 @@ export class PlaythroughService {
     private storePlaythrough(isNewPlaythrough: boolean): void {
       var playthroughId = (
         isNewPlaythrough ? null : this.playthrough.playthroughId);
-      var promise = this.http.post(this.getFullPlaythroughUrl(), {
-        playthrough_data: this.playthrough.toBackendDict(),
-        issue_schema_version: ServicesConstants.CURRENT_ISSUE_SCHEMA_VERSION,
-        playthrough_id: playthroughId
-      }).toPromise();
+      var promise = this.playthroughBackendApiService.storePlaythrough(
+        this.playthrough, ServicesConstants.CURRENT_ISSUE_SCHEMA_VERSION,
+        playthroughId);
       if (isNewPlaythrough) {
-        promise.then((
-            response: {
-                // eslint-disable-next-line camelcase
-                playthrough_stored: boolean, playthrough_id: string }) => {
-          if (response.playthrough_stored) {
+        promise.then((response) => {
```

Comment:
Please don't wrap single-argument arrow functions in parentheses, just use `response => { ... }`.

Reply:
done